infra/proposer.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"io"
 
+	log "github.com/sirupsen/logrus"
+
```

Comment:
remove empty line

Reply:
the empty line is added by vs code automatic.